app/src/main/res/values/strings.xml

```diff
@@ -13,6 +13,7 @@
     <string name="menu_stop">Stop</string>
     <string name="menu_refresh">Refresh</string>
     <string name="menu_export_database">Export database</string>
+    <string name="menu_export_csv">Export CSV</string>
```

Comment:
Mind updating the text here to `Export CSV (SiDiary format)` otherwise it may confuse people why its so sparse on information